tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/metrics/TkmsMetricsTemplate.java

```diff
@@ -34,7 +34,9 @@
 @Slf4j
 public class TkmsMetricsTemplate implements ITkmsMetricsTemplate, InitializingBean {
 
-  public static final String GAUGE_LIBRARY_INFO = "tw_library_info";
```

Comment:
But is it going to create issues with older version of SB?

Reply:
Actually, now that I think about it, it should not

Reply:
Doesn't look like it, already tested with 3.2 as well. 

Micrometer has been supporting both underscore and dot. 

This one looks more like a bug from their end cuz even now they support underscore just that this doesn't get properly converted to an info type.